scopes/ui-foundation/use-box/menu/menu.module.scss

```diff
@@ -1,8 +1,14 @@
+@import '~@teambit/ui-foundation.ui.constants.z-indexes/z-indexes.module.scss';
```

Comment:
can be removed from this PR 
Will be fixed in #6714 

https://github.com/teambit/bit/pull/6714/files#diff-e91ae23f82a550ba9ccc73a109410e51b516d491e817bf7e7a72387bfbaa9dc6